lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Do logics in this file work if the host page has a non-zero scroll offset? I think scroll offset doesn't affect clientX/Y, but it will affect the position of the cursor relative to the elements.

Reply:
Yes, methods that only use offsetX/Y are relative to the element, so the scroll position does not matter.

The one that requires scroll handling is the pageX/Y one, because that gives us positions from the top left position of the page (with all its scroll position added)